spec/spec_helper.rb

```diff
@@ -1,4 +1,5 @@
 # encoding: utf-8
+require "clamp"
```

Comment:
I dont think this library is needed for all the tests, I think this should only required on the corresponding file.